### PR TITLE
Update Habr

### LINKED ISSUE #5596: Update Habr icon
**Brand Name:** Habr
**Website:** https://habr.com/
**Official resources for icon and color:**

Following #5555, it looks like we need to update the icon. I do not find any usage of the squared `h` anymore.

This is the "icon" they use everywhere (favicon and social media)
image

Unfortunately, I did not find any vector versions for now.

### PR DESCRIPTION
**Pull request by PeterShaggyNoble**
Habr

**Issue:** Resolves #5596 
**Alexa rank:** [~1.4k](https://www.alexa.com/siteinfo/habr.com)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with ~SVGO or~ SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Icon from [this SVG](https://kiosk.habr.com/images/habr.svg) found at [source](https://kiosk.habr.com/). Impossible to tell here which parts of the mess are intentional and which, if any, are not. It also proved pretty impossible to remove any duplicate points; removing some would bring back another that I'd previously removed and removing others would distort other parts of the "shape". The couple I _did_ manage to remove, SVGOMG then brought back 🙄 So, fingers crossed, this works out.

For the colour: 🤷🏻‍♂️ Your guess is as good as mine. They use _so_ many different shades of that colour. I just went with the one from the "Sign Up" button in the header at random.

### CODE REVIEW

**Review by adamrusted — APPROVED** [review]
Wow, that icon looks _awful_ to work with.
Yours matches the source perfectly, but I'll hold off merging for now so another @simple-icons/maintainers can okay the hex code too.

**Review by service-paradis — APPROVED** [review]
Really good job on this one @PeterShaggyNoble! 💯 
There are so much blue shades on this page, really hard to choose which one is the brand color. I'm ok with the current choice and it could be easily updated in another release if we find some official color eventually.

**Pull request merged**
